dev/gen

```diff
@@ -5,6 +5,18 @@ cd "$(dirname "$0")/.."
 
 if [[ -e gen ]]; then
     rm -r gen
-    mkdir -p gen/py
 fi
+mkdir -p gen/go gen/ts gen/py
 dev/buf generate .
+
+# Unpack Python protobuf zip and set up package structure
+if [[ -f gen/py/py.zip ]]; then
+    cd gen/py
+    unzip -q py.zip
+    rm py.zip
```

Comment:
Removing the `.zip` in `buf.gen.yaml` will save these as uncompressed. No need to zip and unzip.